tests/test_integration.py

```diff
@@ -328,6 +328,25 @@ def test_get(self, soco):
         assert sorted(info.keys()) == self.info_keys
 
 
+class TestGetCurrentMediaInfo:
+    """Integration test for the get_current_media_info method."""
+
+    info_keys = sorted(
+        [
+            "uri",
+            "channel",
+        ]
+    )
+
+    def test_get(self, soco):
+        """Test that the return value is a dictinary and contains the expected
```

Comment:
Minor typo: 'dictinary'.

Reply:
Thanks. I fixed that, also for the `TestGetCurrentTrackInfo` where I have copied from :-)